### PR TITLE
feat: add logs retrieval to ship cli

### PR DESCRIPTION
**Pull request by douglas-reid**
This adds a command to the `ship` CLI to support logs retrieval.

Users can use it in conjunction with `jq` to retrieve and parse logs in their workspaces (filtering by package/instance/version/path).

Example usage:

```bash
ship logs --workspace my-prod-workspace  --path '/generate'  | jq '.entries[].message'
```

This should allow developers to quickly access pertinent logs for their packages without having to visit the web UI.

### CODE REVIEW

File: src/steamship/cli/cli.py [hunk 1]
```diff
@@ -26,9 +28,10 @@ def cli():
     pass
 
 
-def initialize():
+def initialize(suppress_message: bool = False):
     logging.root.setLevel(logging.FATAL)
-    click.echo(f"Steamship PYTHON cli version {steamship.__version__}")
+    if not suppress_message:
+        click.echo(f"Steamship PYTHON cli version {steamship.__version__}")
```
Comment (eob) [inline src/steamship/cli/cli.py:34]: Oh nice - for jq support I assume

**Review by eob — DISMISSED** [review]
This is really awesome -- we should totally launch this on Twitter when we cut a new version!!

**Review by dkolas — DISMISSED** [review]
Looks good - are you missing a line for adding it to the command group?

File: src/steamship/cli/cli.py [hunk 2]
```diff
@@ -116,6 +119,71 @@ def deploy():
...
+        client = Steamship(workspace=workspace)
+    except SteamshipError as e:
+        raise click.UsageError(message=e.message)
+
+    click.echo(json.dumps(client.logs(offset, number, package, instance, version, request_path)))
+
+
 cli.add_command(login)
 cli.add_command(deploy)
 cli.add_command(deploy, name="it")
```
[the remark this answers is not in the document]
Comment (douglas-reid) [reply src/steamship/cli/cli.py:189]: i was indeed. lost in a rebase. restored.

**Pull request merged**